investigation-service/src/test/resources/rawDataFiles/ObservationNotificationIds.json

```diff
@@ -0,0 +1,59 @@
+[
```

Comment:
This was removed in #83. The tests for ObservationNotificationIds is now mapped for InvestigationObservationIds.

Reply:
Yes